demos/README.md

```diff
@@ -24,9 +24,9 @@ app.get('/', (req, res) => res.send('Hello world'));
 app.listen(port, () => console.log('Application started'));
 ```
 
+* In the terminal window, run the command `node app.js` to run the application .
```

Comment:
It one or the other. Can't run both or we're going to be crowding the port 3000.

Reply:
May want to add a "Run of the following" or an "OR" in there.

Reply:
Hey, actually what I'm trying to say is that the command to run the hello world app, is missing in the documention of demos [(Here)](https://github.com/MaximRouiller/msp001/blob/master/demos/README.md#demo-1---hello-world) just before the last step.
As a new learner doesn't know how to run the app for the first time. So I think that the step should be there for its guidance.